feconf.py

```diff
@@ -414,6 +414,11 @@ def get_empty_ratings():
 # with the Mailgun domain name (ending with mailgun.org).
 MAILGUN_DOMAIN_NAME = None
 
+# NOTE TO RELEASE COORDINATORS: Replace this with the correct ElasticSearch Host
+# and Port when switching to prod server.
+ES_HOST = 'localhost'
+ES_PORT = 9200
```

Comment:
Should we include the changes to these in the release scripts?

Reply:
yeah. these values are read in by `elastic_search_services.py` to determine which server to connect to.